src/action/type.js

```diff
@@ -0,0 +1,20 @@
+export default {
+  GET_ALL_PRODUCTS:'get_all_products',
```

Comment:
Not required but it can be helpful to keep your types in alphabetical order, especially as the types list grow in size.